### PR TITLE
Support suffix exception

### PR DESCRIPTION
**Pull request by ryansteakley**
Issue #, if available:
https://github.com/aws-controllers-k8s/community/issues/840

Description of changes:
Add support for suffix exception checking. Also added Sagemaker tests for prefix and suffix exceptions.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pkg/generate/code/check.go [hunk 1]
```diff
@@ -45,6 +46,10 @@ func CheckExceptionMessagePrefix(
 			return fmt.Sprintf("&& strings.HasPrefix(awsErr.Message(), \"%s\") ",
 				*excConfig.MessagePrefix)
 		}
+		if ok && excConfig.MessageSuffix != nil {
```
Comment (RedbackThomson) [inline pkg/generate/code/check.go:49]: nit: Replace the `if ok` in each of these conditional blocks with an early return at the top:
```go
if !ok {
  return ""
}
```

**Review by RedbackThomson — APPROVED** [review]
Yeah, really like these changes!

File: pkg/generate/code/check.go (hunk 1 above)
Comment (ryansteakley) [reply pkg/generate/code/check.go]: replaced

File: pkg/generate/code/check.go [hunk 2]
```diff
@@ -24,27 +24,35 @@ import (
...
-// If message_prefix field was not specified for this HTTP code in generator
+// If message_prefix/message_suffix field was not specified for this HTTP code in generator
 // config, we return an empty string
 //
 // Sample Output:
 //
 // && strings.HasPrefix(awsErr.Message(), "Could not find model")
-func CheckExceptionMessagePrefix(
+// && strings.HasSuffix(awsErr.Message(), "does not exist.")
+func CheckExceptionMessageAffix(
```
Comment (jaypipes) [inline pkg/generate/code/check.go:37]: We could also just call this `CheckExceptionMessage` :)

**Review by jaypipes — APPROVED** [review]
Great stuff @ryansteakley :)

**Review by jaypipes — APPROVED** [review]

**Pull request merged**